### PR TITLE
Feature/vaccinecer 2186 log key identifier during creation

### PR DESCRIPTION
**Pull request by haraldloesing**

### CODE REVIEW

File: src/main/java/ch/admin/bag/covidcertificate/web/controller/test/TestController.java [hunk 1]
```diff
@@ -48,17 +49,22 @@ public List<SigningInformation> testSigningInformationConfiguration(
...
 
             for (SigningInformation signingInformation : signingInformationList) {
                 try {
                     var messageBytes = UUID.randomUUID().toString().getBytes();
-                    var signatureBytes = signingClient.createSignature(messageBytes, signingInformation);
-                    if (signingInformation.getCertificateAlias() != null && !signingInformation.getCertificateAlias().isBlank()) {
+                    var signatureBytes = signingClient
+                            .createSignature(messageBytes, SigningInformationMapper.fromEntity(signingInformation));
+                    if (signingInformation.getCertificateAlias() != null &&
+                            !signingInformation.getCertificateAlias().isBlank()) {
```
Comment (mofobo-bit) [inline src/main/java/ch/admin/bag/covidcertificate/web/controller/test/TestController.java:61]: import org.apache.commons.lang3.StringUtils;

StringUtils.isNotBlank(signingInformation.getCertificateAlias())

**Review by mofobo-bit — CHANGES_REQUESTED** [review]

File: src/main/java/ch/admin/bag/covidcertificate/client/signing/SigningInformationDto.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    ) {
+        this.certificateType = certificateType;
+        this.code = code;
+        this.alias = alias;
+        this.certificateAlias = certificateAlias;
+        this.validFrom = validFrom;
+        this.validTo = validTo;
+    }
+
+    public static class SigningInformationDtoBuilder {
```
[the remark this answers is not in the document]
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/client/signing/SigningInformationDto.java:39]: Nope, as the Setter is invoked after creation. The Attribute is a derived Attribute calculated during the signing process.

File: src/main/java/ch/admin/bag/covidcertificate/web/controller/test/TestController.java (hunk 1 above)
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/web/controller/test/TestController.java]: Done

File: src/main/java/ch/admin/bag/covidcertificate/api/response/CovidCertificateResponseEnvelope.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class CovidCertificateResponseEnvelope {
+    private CovidCertificateCreateResponseDto responseDto;
+    private String usedKexIdentifier;
```
[the remark this answers is not in the document]
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/api/response/CovidCertificateResponseEnvelope.java]: Done

**Review by iris-hunkeler — APPROVED** [review]

**Pull request merged**